wallet/src/de/schildbach/wallet/ui/send/SendCoinsActivity.java

```diff
@@ -167,4 +167,9 @@ public boolean onOptionsItemSelected(final MenuItem item) {
 
         return super.onOptionsItemSelected(item);
     }
+
+    @Override
+    protected void onActivityResult(int requestCode, int resultCode, Intent data) {
+        super.onActivityResult(requestCode, resultCode, data);
+    }
```

Comment:
Do we need this?

Reply:
The reason why I don't think we need this is because this method is only called the super method from the base class.  Isn't that the same as not having this method at all?